contrib/singularity/prepare-solvcon-dev.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# Usage:
+#   source <this script>
+#   or
+#   source <this script> <your-project-folder>
+#
+SOLVCON_PROJECT=${1:-${HOME}/work-my-projects/solvcon}
+SCSRC="${SOLVCON_PROJECT}/solvcon"
+SCSRC_WORKING="${SOLVCON_PROJECT}/solvcon-working/"
+MINICONDA_DIR="${SCSRC_WORKING}/miniconda"
+
+export PATH="${SCSRC}:${MINICONDA_DIR}/bin/:${PATH}"
+
+mkdir -p ${SCSRC_WORKING}
+
+# fetch SOLVCON source
+git clone https://github.com/solvcon/solvcon.git ${SCSRC}
+
+# prepare miniconda
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ${SCSRC_WORKING}/miniconda.sh
+
+bash ${SCSRC_WORKING}/miniconda.sh -b -p ${MINICONDA_DIR}
+conda config --set always_yes yes
+conda update -q conda
+
```

Comment:
How about removing the downloaded files here, so that the built singularity image doesn’t contain them and bloat?